app/Http/Controllers/Admin/ArtistController.php

```diff
@@ -82,14 +57,7 @@ public function edit(Artist $artist)
         ]);
     }
 
-    /**
-     * Update the specified resource in storage.
-     *
-     * @param  App\Http\Requests\CreateArtist  $request
-     * @param App\Models\Artist  $artist
-     * @return \Illuminate\Http\Response
-     */
-    public function update(CreateArtist $request, Artist $artist)
+    public function update(CreateArtist $request, Artist $artist): RedirectResponse
```

Comment:
We want to add data and we haven't used a repository pattern for the grab or insert data into database.
so it may not be possible to change it at this time.
But still i've updated the code